cognite/powerops/custom_modules/power_model_v1/data_models/views/shop_result/PriceProdCase.view.yaml

```diff
@@ -0,0 +1,46 @@
+#NB! This view is used in the frontend to pair a price/prod pair with a scenario and link it to the multiscenariobidmatrix
+space: '{{powerops_models}}'
+externalId: PriceProdCase
+name: PriceProdCase
+version: '{{version}}'
+description: The output price/prod pair of a shop run linked to the scenario that was used to produce it. This is used in the frontend to pair a price/prod pair with a scenario and link it to the multiscenariobidmatrix
+filter:
+  and:
+    - hasData:
+      - type: container
+        space: '{{powerops_models}}'
+        externalId: SHOPResult
```

Comment:
This one could just as well use FunctionData container, I think? Are there advantages to using the ShopResult container for this type/view?